lti_redirect/utils/utils.py

```diff
@@ -0,0 +1,4 @@
+def check_email_parameter(launch_data):
+    if "email" in launch_data:
+        return launch_data["email"]
+    return None
```

Comment:
I had to add this check incase of `View as student` or Test Student Canvas is not sending any Email. So I added a check to skip the exception. 
![Screenshot 2024-09-03 at 3 42 21 PM](https://github.com/user-attachments/assets/dbff53c3-1f4d-412e-86ea-693eb3910a84)

Reply:
I changed as you suggested and with this it is implicitly returning None. I change a few variables like where the course payload is not from LTI custom variables. Thanks!